anna-sala/challenges/array-methods/test/reduce.test.js

```diff
@@ -0,0 +1,55 @@
+//Does not pass the test!
+
+function annaReduce (array, reduce){
+    let accumulator=0;
+    for (let i=0; i<array.length; i++){
+        accumulator += reduce(accumulator, array[i]);
+   
+    }
+    return true;
+}
```

Comment:
Ojo, que reduce no devuelve true/false

Reply:
no esta bien hecho, lo he comentado en la primera linea diciendo que esto es lo que he intentado. Estaba cogiendo map and find e intentando conbinarlas